app/guid-file/template.hbs

```diff
@@ -1,281 +1,253 @@
-{{page-title this.model.file.name}}
-<div data-analytics-scope='File detail' class='container'>
-    <QuickfileNav @user={{this.model.user}} @onQuickfiles={{false}} />
-    <div data-analytics-scope='Title bar' local-class='TitleBar'>
-        <div data-analytics-scope='Title' local-class='TitleBar__title'>
-            <h2 data-test-file-title-header>
-                {{this.model.file.name}}
-                <BsButton
-                    data-analytics-name='Version'
-                    local-class='TitleBar__version-link'
-                    @bubble={{true}}
-                    @type='link'
-                    @onClick={{action 'changeView' 'revision'}}
-                >
-                    {{t 'file_detail.version.title' versionNumber=this.mfrVersion}}
-                </BsButton>
-            </h2>
+<OsfLayout @backgroundClass={{local-class 'bg-class'}} as |layout|>
+    <layout.main local-class='FileDetail_main'>
+        <div class='row'>
+            <div class='col-lg-11'>
+                <div local-class='FileDetail_registration'>
+                    <BsButton
+                        data-test-registration-link
+                        data-analytics-name='Registration'
+                        local-class='FileDetail__registration-link'
+                        @bubble={{true}}
+                        @type='link'
+                        @onClick={{'#'}}
+                    >
+                        <h5>
+                            {{this.model.file.provider}}
+                        </h5>
+                    </BsButton>
+                </div>
+            </div>
         </div>
-        <div data-analytics-scope='buttons' local-class='TitleBar__buttons'>
-            <div class='pull-right'>
-                <div class='btn-toolbar m-t-md'>
-                    {{#if this.canDelete}}
-                        <div class='btn-group m-l-xs m-t-xs'>
+        <div class='row' local-class='FileDetail_fileNameButtons'>
+            <div class='col-sm-10'>
+                <h3>
+                    <strong>
+                        {{this.model.file.name}}
+                    </strong>
```

Comment:
I think it would be better to add a class to this rather than make it `strong` to give us more control over the display.

Reply:
Ok, I've updated the SCSS file to do so, adding the 'font-weight: bold' attribute instead.